dhis-2/dhis-api/src/main/java/org/hisp/dhis/common/QueryOperator.java

```diff
@@ -47,6 +47,9 @@
     LE( "<=" ),
     LIKE( "like" ),
     IN( "in", true ),
+    // Tracker new operators
```

Comment:
Should we change this to

`// Tracker specific`

as done below for analytics. These will not stay `new` for long 😋 

Can this cause problems somewhere? I mean we add operators in here that are only supported by tracker. Is the rest of the codebase going to ignore these safely 🤷🏻 Are we assuming since thats what we do below for analytics it should also be fine for tracker.

Reply:
Good question Ivo. I dont think it will cause problems elsewhere. I dont see any obvious gaps. So even if analytics end up using the new operators, it should work in theory. I have not tried it myself though.